src/aten/DilatedMaxPool2d.cpp

```diff
@@ -0,0 +1,53 @@
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+
+#include <aten/sycl/DilatedMaxPool2d.h>
+
+namespace at {
+
+Tensor& XPUNativeFunctions::max_pool2d_with_indices_backward_out(
```

Comment:
Lack of meta func,
<img width="749" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/b4845ca2-dbfb-48d2-aea6-5d930cfa3ba1">